examples/global_tables/main.tf

```diff
@@ -0,0 +1,84 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+provider "aws" {
+  alias  = "euwest2"
+  region = "eu-west-2"
+}
+
+#########################
+# Supporting Resources
+#########################
+resource "aws_kms_key" "primary" {
```

Comment:
Is it worth it to use the example to show how to create a KMS key within the module?

Reply:
In this case it was worth because default key is not available is replica region